Demo_Project/src/Component/settings/Settings.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import useStyles from './SettingsStyles';
+import { TextField, Button, Typography } from '@material-ui/core';
+const Settings =()=>{
+    const classes=useStyles()
+    let username="reem"
+    let email="reem2gmail.com"
```

Comment:
pass these (email, username) as props

Reply:
email and password i am going to bring it from backend so should i use them as props?